src/QueryBuilder/QueryBuilder/QueryBuilderModal/QueryBuilderModal.test.js

```diff
@@ -155,8 +155,12 @@ describe('QueryBuilderModal', () => {
 
     await waitFor(() => {
       expect(runQuery).toBeEnabled();
-    });
+    }, { timeout: 5500 });
```

Comment:
It's required because default timeout for waitFor - 1000ms, and we are pulling every 5000ms 
https://testing-library.com/docs/dom-testing-library/api-async/